packages/react-jss/src/utils/sheets-meta.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import type {StyleSheet} from 'jss'
+import type {StaticStyles} from '../types'
+
+interface SheetMeta {
+  styles: StaticStyles;
+  dynamicStyles: StaticStyles;
+  dynamicRuleCounter: number;
+}
+
+const sheetsMeta = new WeakMap<StyleSheet, SheetMeta>()
+
+function getMetaForSheet(sheet: StyleSheet) {
+  return sheetsMeta.get(sheet)
+}
+
+function addMetaForSheet(sheet: StyleSheet, meta: SheetMeta) {
+  sheetsMeta.set(sheet, meta)
+}
+
+export {getMetaForSheet, addMetaForSheet}
```

Comment:
if you use  named functions you could as well export them in place. Our convention though still is to not use named functions at all. It avoids a number of issues.